### PR TITLE
Add FireDragon profile

### PR DESCRIPTION
**Pull request by dr460nf1r3**
Adding a profile for [FireDragon](https://forum.garudalinux.org/t/firedragon-librewolf-fork/5018), a Librewolf fork.

### CODE REVIEW

File: etc/profile-a-l/firedragon.profile [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+# Add the next lines to your firedragon.local if you want to use the migration wizard.
+#noblacklist ${HOME}/.mozilla
+#whitelist ${HOME}/.mozilla
+
+# FireDragon requires a shell to launch on Arch. We can possibly remove sh though.
+# Add the next line to your firedragon.local to enable private-bin.
+#private-bin bash,dbus-launch,dbus-send,env,firedragon,python*,sh,which
+# Add the next line to your firedragon.local to enable private-etc. Note
+# that private-etc must first be enabled in firedragon-common.local.
```
Comment (rusty-snake) [inline etc/profile-a-l/firedragon.profile:25]: There is no firedragon-common.profile.
```suggestion
# that private-etc must first be enabled in firefox-common.local.
```

File: etc/profile-a-l/firedragon.profile [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+#private-etc firedragon
```
Comment (rusty-snake) [inline etc/profile-a-l/firedragon.profile:26]: Does this directory exist?

File: etc/profile-a-l/firedragon.profile [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Firejail profile for FireDragon
+# Description: Librewolf fork with enhanced KDE integration
+# This file is overwritten after every install/update
+# Persistent local customizations
+include firedragon.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.cache/firedragon
+noblacklist ${HOME}/.firedragon
```
Comment (rusty-snake) [inline etc/profile-a-l/firedragon.profile:10]: Add `blacklist`s in disable-programs.inc

**Review by rusty-snake — COMMENTED** [review]
Looks good so far. Some nitpicks below. Also add it to firecfg.config.

File: etc/profile-a-l/firedragon.profile (hunk 2 above)
Comment (dr460nf1r3) [reply etc/profile-a-l/firedragon.profile:26]: Which one do you mean?

File: etc/inc/disable-programs.inc [hunk 4]
```diff
@@ -228,6 +228,7 @@ blacklist ${HOME}/.config/evince
 blacklist ${HOME}/.config/evolution
 blacklist ${HOME}/.config/falkon
 blacklist ${HOME}/.config/filezilla
+blacklist ${HOME}/.config/firedragon
```
Comment (rusty-snake) [inline etc/inc/disable-programs.inc:231]: `${HOME}/.config/firedragon`?
`${HOME}/.firedragon`?

Comment (vnepogodin) [reply etc/inc/disable-programs.inc:231]: only `${HOME}/.firedragon` needed

File: etc/profile-a-l/firedragon.profile (hunk 2 above)
Comment (rusty-snake) [reply etc/profile-a-l/firedragon.profile:26]: /etc/firedragon

Comment (dr460nf1r3) [reply etc/profile-a-l/firedragon.profile:26]: Nope, in fact there is not. So this part can be removed I guess. I mostly took Librewolf as template for this profile which has this in there.

**Review by glitsj16 — APPROVED** [review]
LGTM

**Comment by rusty-snake** [comment]
Merged, thanks.

**Comment by dr460nf1r3** [comment]
:)

**Pull request merged**